plugins/premium/tinymcespellchecker.md

```diff
@@ -204,19 +204,32 @@ tinymce.init({
 });
 ```
 
-### `spellchecker_whitelist`
+### `spellchecker_ignorelist`
```

Comment:
Another thought... this will break any bookmarks to this section. So do we need to add something like this?
```
<a class="anchor" id="spellchecker_whitelist">
### `spellchecker_ignorelist`
```